python/ray/tune/syncer.py

```diff
@@ -60,8 +53,8 @@ def set_sync_periods(sync_config: "SyncConfig"):
             "'TUNE_CLOUD_SYNC_S' is deprecated. Set "
             "`cloud_sync_period` via tune.SyncConfig instead.")
         CLOUD_SYNC_PERIOD = env_integer(key="TUNE_CLOUD_SYNC_S", default=300)
-    NODE_SYNC_PERIOD = int(sync_config.node_sync_period)
-    CLOUD_SYNC_PERIOD = int(sync_config.cloud_sync_period)
+    NODE_SYNC_PERIOD = int(sync_config.sync_period)
+    CLOUD_SYNC_PERIOD = int(sync_config.sync_period)
```

Comment:
If they are the same all the time, why keep both NODE_SYNC_PERIOD and CLOUD_SYNC_PERIOD?

Reply:
Good point!